src/main/java/za/ac/cput/factory/ProductFactory.java

```diff
@@ -3,21 +3,25 @@
 import za.ac.cput.domain.Product;
 import za.ac.cput.util.ProductHelper;
 
-//Is valid imageURL needs to be checked
```

Comment:
Did you check if the validImageURL helper works

Reply:
No let me test that now .

Reply:
sure it works, nice